docs/DeveloperGuide.md

```diff
@@ -679,7 +671,7 @@ name with that assignment.
 The following activity diagram summarizes what happens when a user executes the done command:
 
 <p align="center">
-  <img src="images/DoneActivityDiagram.png" width="250" />
+  <img src="images/DoneActivityDiagram.png" width="600" height="550" />
```

Comment:
so some activity diagrams have width 500 some have 600, is it intentional?

Reply:
Yes! Some activity diagrams have longer sentences, so a bigger width will make the words easier to read!